lesson08/index.js

```diff
@@ -0,0 +1,70 @@
+const imageArray = [
+    {
+        img: "1.png", 
+        alt:"画像1", 
+        text: "ブックマーク"
+    }, 
+    {
+        img: "2.png", 
+        alt:"画像2", 
+        text: "メッセージ"
+    }
+];
+
+const ul = document.getElementById('js-list');
+
+const createLoading = () => {
+    const li = document.createElement("li");
+    const loadingImg = document.createElement('img');
+
+    li.style.listStyle = "none";
+    li.id = "js-loading";
+    loadingImg.src = 'loading-circle.gif';
+    ul.appendChild(li).appendChild(loadingImg);
+};
+
+const removeLoading = () => {
+    const li = document.getElementById('js-loading');
+
+    ul.removeChild(li);
+};
+
+const addList = (val) => {
+    const fragment = document.createDocumentFragment();
+    for (let i = 0; i < val.length; i++) {
+        const li = document.createElement('li');
+        const img = document.createElement('img');
+    
+        li.textContent = val[i].text;
+        img.src = val[i].img;
+        img.alt = val[i].alt;
+    
+        fragment.appendChild(li).appendChild(img);
+    }
+
+    ul.appendChild(fragment);
+};
+
+/**
+ * Promiseを実行し、rejectの値を返す関数
+ * @return {promise object} 
+ */
+const fetchData = () => {
+    createLoading();
+    return new Promise( (reject) => {
+        setTimeout(() => {
+            reject(new Error("ERROR!"));
```

Comment:
MUST
I don't think reject has been executed. If it is running, I think the process will be moved to catch and the error will be displayed in console.
Please check.

Reply:
Thank you for review!
There were unnecessary functions and arrays that were not displayed in the console.
After deleting them, it was displayed successfully.
[161b612](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/8/commits/161b61260860a77cf54b6a8754e719ae1fa7b1c9)

Reply:
I'm sorry, I didn't communicate that well.

Currently, resolve is being executed instead of reject.
There are two arguments for promise, resolve and reject, but reject must be specified as the second argument

Reply:
I'm sorry about that too.
Added resolve as the first argument.
[710c41b](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/8/commits/710c41b5ab6b8e1f739eb6fe1d765f227c3fd641)